src/components/NavigationBar/style.css

```diff
@@ -1,8 +1,6 @@
 @import (less) "../Theme/constants.css";
 
 .navigation {
-  height: 32*@unit;
-  padding: 16*@unit 16*@unit 0;
```

Comment:
so what is the different between `.navigation` and `navbar`? also the height and padding is also defined below inside of a media query. won't that be a problem?

Reply:
Sorry, navbar is just short for navigation bar. I should probably call the variables `@navigation-...` to avoid confusion. I will make sure the media query uses the variables too.